src/lib/services/account-abstraction/tokens.ts

```
@@ -0,0 +1,305 @@
+	return (Object.entries(PAYMENT_TOKENS_MULTI_NETWORK) as [string, PaymentToken[]][])
+		.filter(([, tokens]) => tokens.some((t) => t.symbol === symbol))
+		.map(([chainId]) => Number(chainId) as SupportedNetworkId);
+}
+
+/**
+ * Get all unique token symbols across all networks
+ */
+export function getAllTokenSymbols(): string[] {
+	const symbols = new Set<string>();
+	Object.values(PAYMENT_TOKENS_MULTI_NETWORK).forEach((tokens) => {
+		tokens.forEach((t) => symbols.add(t.symbol));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)